modules/ui/src/main/resources/lucuma-css/lucuma-ui-table.scss

```diff
@@ -4,6 +4,8 @@
       border-collapse: collapse;
       border-spacing: 0;
       table-layout: fixed;
+      position: relative;
```

Comment:
If I understand correctly, we will not be using Prime at all to style tables now. Do we we want to change the `pl` prefix then? (Which stands for `prime`-`lucuma`). Maybe just `l`?

Reply:
Actually, we are using primereact to style the tables, but just the theming stuff that is in the theme files. The CSS that prime moved to the components is more for layout that is independent of theming. We only made use of a very small amount of that. (these 2 lines)